maps/torch/datums/uniforms_fleet.dm

```diff
@@ -309,15 +307,15 @@
 	name = "Fleet supply CO"
 	min_rank = 11
 
-	utility_extra = list(
-				/obj/item/clothing/under/solgov/utility/fleet/combat/supply,
-				/obj/item/clothing/head/beret/solgov/fleet/command,
-				/obj/item/clothing/head/beret/solgov/fleet/supply,
-				/obj/item/clothing/head/ushanka/solgov/fleet,
-				/obj/item/clothing/suit/storage/hooded/wintercoat/solgov/fleet,
-				/obj/item/clothing/head/soft/solgov/fleet,
-				/obj/item/clothing/gloves/thick/duty/solgov/sup
-			)
+		utility_extra = list(/obj/item/clothing/under/solgov/utility/fleet/officer/command,
```

Comment:
This is where the checks fail (line 308). Inconsistent indentation leads to a compilation error.